src/appkit.cs

```diff
@@ -904,6 +904,22 @@ interface NSApplicationDelegate {
 		void UserDidAcceptCloudKitShare (NSApplication application, CKShareMetadata metadata);
 #endif
 	}
+
+	[Mac (10, 12, 1)]
+	[Category]
+	[BaseType (typeof(NSApplication))]
+	interface NSApplication_NSTouchBarCustomization
+	{
+		[Export ("isAutomaticCustomizeTouchBarMenuItemEnabled")]
+		bool GetAutomaticCustomizeTouchBarMenuItemEnabled ();
```

Comment:
Here and a few other places we had to convert a property to a get / set method since we don't have extension properties, and we're a category.

Would it be better to inline these in the specific class (to get new {get; set;}?